### ISSUE TITLE
Updating a SCORM from "Learning paths" does not work

### ISSUE THREAD

**Issue by juan-cortizas-ponte** [chamilo/chamilo-lms#4429]
**Describe**

The update SCORM feature, on "Learning paths" list, does not work [since this commit](https://github.com/chamilo/chamilo-lms/commit/28fee3c73168c67a024a526f4d37c422633c4270) that adds the bigupload lib. Chamilo does not create anymore a folder with the zip original name at app/courses/{course_code}/scorm/, but a folder with a random name. So when you try to update the SCORM, the name of the original zip does not match the folder name that Chamilo creates and the update process fail.

**To Reproduce**

1. Upload a new SCORM zip file
2. Click on the update icon for the scorm uploaded
3. Select again the SCORM zip file

**Expected behavior**

The behavior when importing a SCORM should be the same as before the commit [28fee3c73168c67a024a526f4d37c422633c4270](https://github.com/chamilo/chamilo-lms/commit/28fee3c73168c67a024a526f4d37c422633c4270).

The folder created when importing the SCORM should have the same name as the zip file

**Screenshots**

The update tool location:
![imagen](https://user-images.githubusercontent.com/48205899/197009600-3fe6637d-b33b-4500-a460-5f341ef7a7fc.png)

Debugging the import of a new SCORM on lp_upload.php, the SCORM original name is "scorm12-120621-271.zip":
![imagen](https://user-images.githubusercontent.com/48205899/197014737-10e51459-4e52-4e37-a49c-5f06b1346a7e.png)

Debugging the import of a new SCORM on scorm.class.php:
![imagen](https://user-images.githubusercontent.com/48205899/197015092-3d039336-43f5-40f4-bde3-4f564d388683.png)

Debugging the import of a new SCORM on scorm.class.php:
![imagen](https://user-images.githubusercontent.com/48205899/197015357-2ccafb7a-890e-44f6-8761-f1b292b84450.png)

Uploading the same SCORM zip file again as an update of the previously imported, path names check fail:
![imagen](https://user-images.githubusercontent.com/48205899/197015987-bc1dcda6-fdd5-4858-a7f4-b62e9df85ecc.png)

Error:
![imagen](https://user-images.githubusercontent.com/48205899/197016789-383db9d9-16ef-41b9-ab7c-fdb441467da1.png)

**Desktop:**
 - OS: Windows 10
 - Browser Firefox 105.0.3 (64-bit) 
 
**Desktop:**
 - OS: Windows 10
 - Browser Chrome 106.0.5249.119 (64 bits)

**Server**
 - OS: Debian GNU/Linux 10 (buster)
 - Version of Chamilo: 1.11.x of 2022-06-22
 - Version of PHP: 7.2.30

**Server**
 - OS: Windows 10
 - Version of Chamilo: 1.11.x of 2022-10-20
 - Version of PHP: 7.2.31

**Comment by christianbeeznest** [comment]
I just added a PR about it https://github.com/chamilo/chamilo-lms/pull/4439 , the scorm folder keeps the name, so it is possible to update now.

**Comment by Coursenligne** [comment]
Strange, as I managed localy and distantly to upload and modify SCORM packages both from the internal Chamilo's authoring tool and an extra vendor tool.
2 things I had to modify thought:
1- In configuration.php there are SCORM parts that are false by default, I turned them to true.
2- In php.ini of my server, I had to change the post_max_size value as it was too low.

There was also a course_import.php file amended. It's mainly used for csv matter, but I uploaded the patch without any trouble.
I hope this helps.
